resources/sts/4.12/hypershift/openshift_hcp_kms_provider_credential_policy.json

```diff
@@ -1,4 +1,5 @@
 {
+    "Sid": "VolumeEncryption",
```

Comment:
This needs to move down to the statement [] section

Reply:
Fixed